src/Round.js

```diff
@@ -0,0 +1,25 @@
+const Turn = require('../src/Turn');
+
+class Round {
+  constructor(deck, turn) {
+    this.deck = deck.deck;
+    this.turns = 0;
+    this.incorrectGuesses = [];
+    this.returnCurrentCard = () => {
+      return this.deck[0];
+    }
+    this.takeTurn = (guess) => {
+      this.turns++;
+      let currentCard = this.returnCurrentCard();
+      let newTurn = new Turn(guess, currentCard);
+      let result = newTurn.evaluateGuess();
+      !result ? this.incorrectGuesses.unshift(currentCard.id) : null;
+      this.deck.shift();
+      return newTurn.giveFeedback(result);
```

Comment:
Is this considered bad practice for SRP? It still technically only does "one thing"